references.md

```diff
@@ -0,0 +1,3 @@
+# References
+
+tbd
```

Comment:
let's not keep adding tbd pages until they are really done. have no idea what 'references' means here.

Reply:
https://docs.microsoft.com/en-us/style-guide/developer-content/reference-documentation in case you wonder what "references" means in documentation